dotnet.cmd

```diff
@@ -0,0 +1,22 @@
+@echo off
+
```

Comment:
nit: unless you want to set environment variables globally on the cmd.exe window that is invoking this script, I will suggest using `setlocal`.